### PR TITLE
GCC GA

### PR DESCRIPTION
**Pull request by get-itips**
@HowlinWolf-92 please kindly review

closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8261

source https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=&searchterms=82332

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/approval-admin.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The first approval created from the Approvals app will trigger the provisioning
 This article describes the Approvals app requirements and roles.
 
 > [!NOTE]
-> This feature hasn't been released to Government Community Cloud (GCC), Government Community Cloud High (GCCH), and Department of Defense (DOD) users yet.
+> This feature hasn't been released to Government Community Cloud High (GCCH), and Department of Defense (DOD) users yet.
```
Comment (JohanFreelancer9) [inline Teams/approval-admin.md:39]:
```suggestion
> This feature hasn't been released to Government Community Cloud High (GCCH) and Department of Defense (DOD) users yet.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@HowlinWolf-92 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**